src/renderers/shaders/ShaderChunk/envmap_fragment.glsl.js

```diff
@@ -38,10 +38,6 @@ export default /* glsl */`
 
 		vec4 envColor = textureCube( envMap, vec3( flipEnvMap * reflectVec.x, reflectVec.yz ) );
 
-	#elif defined( ENVMAP_TYPE_CUBE_UV )
```

Comment:
This shader chunk is used only basic, lambert, phong, toon, and gouraud, none of which support ENVMAP_TYPE_CUBE_UV